### PR TITLE
:recycle: (fish) use eza in fish as well , removes lsd

### PR DESCRIPTION
**Pull request by rubiin**
# Pull Request

As we are using `eza` for zsh and `lsd` for fish , it would be better to use `eza` for both as it saves us from trouble of maintaining configs for both

## Description

Please read these instructions and remove unnecessary text.

- Try to include a summary of the changes and which issue is fixed.
- Also include relevant motivation and context (if applicable).
- List any dependencies that are required for this change. (e.g., packages or other PRs)
- Provide a link if there is an issue related to this pull request. e.g., Fixes # (issue)
- Please add Reviewers, Assignees, Labels, Projects, and Milestones to the PR. (if applicable)

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [x] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [x] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [ ] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [ ] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [ ] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

## Screenshots
Using `eza` on fish shell

image

(if appropriate)

### CODE REVIEW

**Comment by kRHYME7** [comment]
difference of both? I  didnt care until now Lol.

**Comment by rubiin** [comment]
> difference of both? I didnt care until now Lol.

They are the same, replacement for ls  . but eza is more feature rich

**Comment by rubiin** [comment]
any updates

**Comment by rubiin** [comment]
any updates on this

**Comment by abenezerw** [comment]
makes sense i guess, he is just saying using eza for both fish and zsh would reduce the need for an additional lsd config since we are using lsd for fish.

**Comment by abenezerw** [comment]
> makes sense i guess, he is just saying using eza for both fish and zsh would reduce the need for an additional lsd config since we are using lsd for fish.

Was looking at an install of this hyprdots, and it actually mentions this in the install. 
IMG_1328

**Comment by rubiin** [comment]
@abenezerw thanks for mentioning that. I will push a fix. Seems I forgot to update a file

File: Configs/.config/lsd/colors.yaml
Comment (kRHYME7) [inline Configs/.config/lsd/colors.yaml:1]: Let's keep the lsd configs for now.

File: Scripts/restore_cfg.lst
Comment (kRHYME7) [inline Scripts/restore_cfg.lst:1]: Let's keep the config for now. It might be useful for others.

File: Configs/.config/fish/config.fish
Comment (kRHYME7) [inline Configs/.config/fish/config.fish:1]: Didn't test, but can you confirm that the flags works like lsd or ls?

**Review by kRHYME7 — CHANGES_REQUESTED** [review]
Please bring back the lsd configuration.

File: Configs/.config/fish/config.fish
Comment (rubiin) [reply Configs/.config/fish/config.fish:1]: yeah . They do

**Pull request merged**